### DISCUSSION TITLE
react-table + react-window refresh data and jump

### DISCUSSION THREAD

**Discussion by linxu** [TanStack/table#2538] [General]
When the table state data of a row is modified, the table scrollbar jumps back to the first row and refreshes the whole table data. Can we only the data of the modified row be refreshed instead of all?

**Comment by gargroh** [comment]
Partial update/re-render of table is currently not possible, but i am sure you can stop that scroll jump

**Comment by linxu** [comment]
@gargroh Do you have any good solutions?
I am scrolling it back to the original position now, when the table is re-rendered.
In this way, i need to record the coordinates of the scroll in real.

**Comment by gargroh** [comment]
not sure tht i have done any special handle ng for tht....once try react
virtual may be it can solve tht issue out of box...
